ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -223,6 +230,29 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
         }
     }
 
+    /// Setting this property  will result in the sheet trying to be as close to this width as possible.
+    /// If the declared width is too large it will roll back to the maximum width
+    @objc open var preferredWidth: CGFloat = 0 {
```

Comment:
Another small thing - are these meant to be subclassed? If they are meant to be set and not subclassed, could remove `open` I think and see what that does to generated code size?

Reply:
Public and open both provide the same binary size. Going to leave it as open remain consistent with the other public properties that support subclassing.